exec/java-exec/src/main/java/org/apache/drill/exec/store/pcap/decoder/TcpHandshake.java

```diff
@@ -0,0 +1,118 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.drill.exec.store.pcap.decoder;
+
+/**
+ * This class is used to record the status of the TCP Handshake. Initially this is used just to determine whether a session is open or closed, but
+ * future functionality could include SYN flood identification, or other hackery with TCP flags.
+ */
+public class TcpHandshake {
+  boolean syn = false;
```

Comment:
Class variables should be private and accessed through getters.

Reply:
I ended up removing all these because they aren't necessary.

Reply:
I guess this should have been done before submitting the PR...

Reply:
After some additional code cleanup, I realized they weren't being used. ;-)
They're removed.